frontend/lib/src/components/widgets/AudioInput/styled-components.ts

```diff
@@ -64,7 +64,10 @@ export const StyledErrorTextSpan = styled.span(({ theme }) => ({
   color: theme.colors.bodyText,
 }))
 
-export const StyledNoMicInputLearnMoreLink = styled.a()
+export const StyledNoMicInputLearnMoreLink = styled.a(({ theme }) => ({
+  color: theme.colors.linkText,
+  textDecoration: "underline",
+}))
```

Comment:
nit: I think we should probably have one top-level styled component to import these general ones from, as `styled.a` is not unique to the AudioInput. What do you think?

Reply:
There isn't really any other styled component yet with the exact same settings. Markdown uses the same settings but has CSS within the markdown-styled component. I will look into links again in a subsequent PR since there are a few more aspects to clean up around link stylings.